### PR TITLE
Portable Ruby cleanup/fixes

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Post-merge review cleanup on https://github.com/Homebrew/brew/pull/17501.

@carlocab suggested variable name change (I agree) and there was a missed version variable change.

### CODE REVIEW

**Review by carlocab — APPROVED** [review]

File: Library/Homebrew/standalone/init.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@
 require "pathname"
 dir = __dir__ || raise("__dir__ is not defined")
 HOMEBREW_LIBRARY_PATH = Pathname(dir).parent.realpath.freeze
-HOMEBREW_PORTABLE_RUBY = Pathname.new(RbConfig.ruby).to_s.include?("/vendor/portable-ruby/").freeze
+HOMEBREW_USING_PORTABLE_RUBY = Pathname.new(RbConfig.ruby).to_s.include?("/vendor/portable-ruby/").freeze
```
Comment (carlocab) [inline Library/Homebrew/standalone/init.rb:32]:
```suggestion
HOMEBREW_USING_PORTABLE_RUBY = RbConfig.ruby.include?("/vendor/portable-ruby/").freeze
```
I think this should just work?

Comment (MikeMcQuaid) [reply Library/Homebrew/standalone/init.rb:32]: @carlocab Will push this to the next PR, incoming shortly 😁

Comment (MikeMcQuaid) [reply Library/Homebrew/standalone/init.rb:32]: @carlocab done in https://github.com/Homebrew/brew/pull/17504

**Comment by Bo98** [comment]
Sorry for the mess up

**Comment by MikeMcQuaid** [comment]
@Bo98 not your fault at all. too many places need updated! very easy to miss.

**Pull request merged**